packages/sentry/src/sentry.ts

```diff
@@ -21,6 +22,50 @@ const sendErrorToSentry = (error: Reportable) => {
   return Sentry.captureException(errorToCapture);
 };
 
+type TReplacements = {
+  [name: string]: string;
+};
+
+const replaceValues = <T>(source: T, replacements: TReplacements): T => {
```

Comment:
I don't think we need a generic here, we can declare `source` to be an `Event`. Maybe then it's easier to deal with the type errors?

Reply:
Nope. It won't simplify things. It it's the typing of Event which makes things hard as it's a lot of optionals and unknowns.